### PR TITLE
Making some small updates to Memory and S.R.CS test projects.

### PR DESCRIPTION
**Pull request by ahsonkhan**
Related to:
https://github.com/dotnet/corefx/pull/23905
https://github.com/dotnet/corefx/pull/20943

cc @KrzysztofCwalina, @stephentoub, @joperezr, @safern

### CODE REVIEW

File: src/System.Memory/tests/Performance/System.Memory.Performance.Tests.csproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <Compile Include="Perf.Span.Clear.cs" />
     <Compile Include="Perf.Span.Fill.cs" />
+    <Compile Include="Perf.Span.IndexOf.cs" />
```
Comment (joperezr) [inline src/System.Memory/tests/Performance/System.Memory.Performance.Tests.csproj]: I suppose you just forgot in a different PR to add this right?

**Comment by joperezr** [comment]
@dotnet-bot test Linux x64 Release Build please
@dotnet-bot test OSX x64 Debug Build please
@dotnet-bot test UWP CoreCLR x64 Debug Build please

(all seem to be unrelated test failures)

File: src/System.Memory/tests/Performance/System.Memory.Performance.Tests.csproj (hunk 1 above)
Comment (ahsonkhan) [reply src/System.Memory/tests/Performance/System.Memory.Performance.Tests.csproj]: Correct.

**Comment by ahsonkhan** [comment]
@dotnet-bot test Linux x64 Release Build

**Comment by ahsonkhan** [comment]
@dotnet-bot test Linux x64 Release Build

**Comment by ahsonkhan** [comment]
@dotnet-bot test Linux x64 Release Build

**Pull request merged**